flocker/control/_model.py

```diff
@@ -199,6 +201,7 @@ class Application(PRecord):
     restart_policy = field(mandatory=True, initial=RestartNever())
     environment = field(mandatory=True, initial=pmap(), factory=pmap,
                         type=PMap)
+    running = field(mandatory=True, initial=True, type=bool)
 
```

Comment:
Also note that in discovery the `running` value is not true by default, but rather is set based on what Docker says state of container is. The default is purely an internal Python API thing, not a semantic default exposed to the HTTP API.